### DISCUSSION TITLE
Tour right/left buttons locations move

### DISCUSSION THREAD

**Discussion by danconde** [azadbolour/eagerwords-community#10] [Ideas]
On web game's [Tour](https://eagerwords.com/tour/play/start.html), the right and left buttons change locations, so it requires users to move cursor at each step. Putting it at same location (maybe at left) will make it easier.

**Comment by azadbolour** [comment]
Agreed that this is an inconvenience. Added to the backlog to be
prioritized as time allows.

Azad
